crates/ruff_python_ast/ast.toml

```diff
@@ -81,38 +81,129 @@ anynode_is_label = "expression"
+    { name = "left", type = "Expr" },
+    { name = "op", type = "Operator" },
+    { name = "right", type = "Expr" }
+]}
+ExprUnaryOp = { fields = [
+    { name = "op", type = "UnaryOp" },
+    { name = "operand", type = "Expr" }
+]}
+ExprLambda = { fields = [
+    { name = "parameters", type = "Box<crate::Parameters>", optional = true },
+    { name = "body", type = "Expr" }
+]}
+ExprIf = { fields = [
+    { name = "test", type = "Expr" },
+    { name = "body", type = "Expr" },
+    { name = "orelse", type = "Expr" }
+]}
+ExprDict = { fields = [
+    { name = "items", type = "DictItem", seq = true },
+]}
+ExprSet = { fields = [
+    { name = "elts", type = "Expr", seq = true }
+]}
+ExprListComp = { fields = [
+    { name = "elt", type = "Expr" },
+    { name = "generators", type = "Comprehension", seq = true }
+]}
+ExprSetComp = { fields = [
+    { name = "elt", type = "Expr" },
+    { name = "generators", type = "Comprehension", seq = true }
+]}
+ExprDictComp = { fields = [
+    { name = "key", type = "Expr" },
+    { name = "value", type = "Expr" },
+    { name = "generators", type = "Comprehension", seq = true }
+]}
+ExprGenerator = { fields = [
+    { name = "elt", type = "Expr" },
+    { name = "generators", type = "Comprehension", seq = true },
+    { name = "parenthesized", type = "bool" }
+]}
+ExprAwait = { fields = [
+    { name = "value", type = "Expr" }
+]}
+ExprYield = { fields = [
+    { name = "value", type = "Expr", optional = true }
+]}
+ExprYieldFrom = { fields = [
+    { name = "value", type = "Expr" }
+]}
+ExprCompare = { fields = [
+    { name = "left", type = "Expr" },
+    # TODO: We don't want this field to be a vec hence the seq is not enabled.
+    { name = "ops", type = "Box<[crate::CmpOp]>"},
+    { name = "comparators", type = "Box<[Expr]>"}
+]}
+ExprCall = { fields = [
+    { name = "func", type = "Expr" },
+    { name = "arguments", type = "Arguments"}
+]}
+ExprFString = { fields = [
+    { name = "value", type = "FStringValue" }
+]}
+ExprStringLiteral = { fields = [
+    { name = "value", type = "StringLiteralValue" }
+]}
+ExprBytesLiteral = { fields = [
+    { name = "value", type = "BytesLiteralValue" }
+]}
+ExprNumberLiteral = { fields = [
+    { name = "value", type = "Number" }
+]}
+ExprBooleanLiteral = { fields = [
+    { name = "value", type = "bool" }
+], derives = ["Default"]}
+ExprNoneLiteral = { fields = [], derives = ["Default"]}
+ExprEllipsisLiteral = { fields = [], derives = ["Default"]}
+ExprAttribute = { fields = [
+    { name = "value", type = "Expr" },
+    { name = "attr", type = "Identifier" },
+    { name = "ctx", type = "ExprContext" }
+]}
+ExprSubscript = { fields = [
+    { name = "value", type = "Expr" },
+    { name = "slice", type = "Expr" },
+    { name = "ctx", type = "ExprContext" }
+]}
+ExprStarred = { fields = [
+    { name = "value", type = "Expr" },
+    { name = "ctx", type = "ExprContext" }
+]}
+ExprName = { fields = [
+    { name = "id", type = "name::Name" },
```

Comment:
> I'm not sure if we refer to `Name` just by it's name and import it in the file or like this:
> 
> ```
> { name = "id", type = "name::Name" },
> ```

I think it's worth ensuring that we can use just `Name` here in the TOML, and importing it at the top of the generated Rust file is a perfectly fine way to do that